### PR TITLE
added two seperate java files/classes for spanish card and suite

### PR DESCRIPTION
**Pull request by gegegenoge7**

### CODE REVIEW

**Comment by marsellusk** [comment]
Great job. This makes it easier for when we want the user to be able to switch between types of decks. Also displaying the spanish card should be simple now.

File: src/main/java/models/Spanish_Deck.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    public Deck1() {
+        value = 50;
+        for(int i = 2; i < 14; i++){
+            deck1.add(new Card(i,Suit1.Bastos));
+            deck1.add(new Card(i,Suit1.Copas));
+            deck1.add(new Card(i,Suit1.Oros));
+            deck1.add(new Card(i,Suit1.Espadas));
+        }
+        //ADD JOKER HERE
```
Comment (marsellusk) [inline src/main/java/models/Spanish_Deck.java]: Overall the code is simple and understandable for the new Spanish Deck class. For add joker here I recommend jokers be a separate entity that we can check. That way once it is in the deck it's not hard to check for it in the other functions. For the user class it is good that you added a deal less than 13 condition. This stops a possible bug where the game can infinitely deal.

File: src/main/java/models/User.java [hunk 2]
```diff
@@ -57,10 +58,20 @@ public void move(int columnNumber, Game g) {
...
+            times_dealt++;
+        }else if (times_dealt = 13) {
+            for (int i = 0; i < 2; i++) {
+                g.addCardToCol(i, g.deck1.deck.get(g.deck1.deck.size() - 1));
+                g.deck1.deck.remove(g.deck1.deck.size() - 1);
+            }
         }
         g.error = " ";
+
+        if
```
Comment (duli4) [inline src/main/java/models/User.java]: Her is a error for if, there are no condition for if.

**Comment by duli4** [comment]
For the Spanish_deck class, we didn't not to create it, because we just can merge all of the suit in Spanish rule to original deck, and when we create Spanish_deck, we just need to call those Spanish suit in deck. It would be more easier.

**Pull request merged**